code/modules/surgery/rig_removal.dm

```diff
@@ -0,0 +1,58 @@
+//Procedures in this file: Unsealing a Rig.
```

Comment:
I am working on a solution to this that I think will have both positive  immersion qualities.